README.md

```diff
@@ -185,7 +185,7 @@ This required Cosign binary to be installed prior to running installation script
 Command:
 
 ```bash
-trufflehog git https://github.com/trufflesecurity/test_keys --only-verified
+trufflehog git https://github.com/trufflesecurity/test_keys --results=verified,unknown
```

Comment:
Including both `verified` and `unknown` introduces people to the concept beyond simply `--only-verified`. Surfacing errors is an important part of using TruffleHog effectively.

If they don't like what they see, they can easily remove `unknown`.